@jkoritzinsky there was no `debian-12-amd64` so I tried `debian-12-helix-amd64` but that won't build. Should I try adding that one? 

Build error: _##[error]Docker-exec executed: `groupadd azure_pipelines_sudo`; container id: `286412e196df2e4f09367ba1a3bcfd146daa7affb610b4a42d45228e59adbf0c`; exit code: `10`; command output: `groupadd: Permission denied.`, `groupadd: cannot lock /etc/group; try again later.`_

--- a/.vsts-ci.yml
+++ b/.vsts-ci.yml
@@ -54,7 +54,7 @@ extends:
       centosStream9:
         image: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream9
       debian12:
-        image: mcr.microsoft.com/dotnet-buildtools/prereqs:debian-12-helix-amd64
+        image: mcr.microsoft.com/dotnet-buildtools/prereqs:debian-12-gcc15-amd64
       fedora41:
         image: mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-41
       ubuntu2204:

--- a/.vsts-pr.yml
+++ b/.vsts-pr.yml
@@ -97,7 +97,7 @@ stages:
       parameters:
         agentOs: Linux
         jobName: Build_Debian_12_Debug_x64
-        container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:debian-12-helix-amd64'
+        container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:debian-12-gcc15-amd64'
         buildConfiguration: Debug
         buildArchitecture: x64
         additionalBuildParameters: '/p:BuildSdkDeb=true'

--- a/test/SdkTests/TestsToSkipLinux.xml
+++ b/test/SdkTests/TestsToSkipLinux.xml
@@ -21,5 +21,8 @@
     <Method Name="Microsoft.NET.Publish.Tests.GivenThatWeWantToPublishAFrameworkDependentApp.It_publishes_with_or_without_apphost"
             Skip="true"
             Reason="Fails on .NET 2.0 and 2.1 so suppressing"/>
-        </SkippedTests>
+    <Method Name="Microsoft.NET.Publish.Tests.GivenThatWeWantToPublishAWebApp.It_should_publish_self_contained_for_2x"
+            Skip="true"
+            Reason="Fails on .NET 2.0 so suppressing"/>
+  </SkippedTests>
 </Tests>

